Detectors/TPC/simulation/src/Baseline.cxx

```diff
@@ -65,11 +65,13 @@ float Baseline::getRandomNoise()
 //______________________________________________________________________________
 float Baseline::getPseudoRealisticNoise(const PadSecPos& padSecPos)
 {
+  return 0.;
```

Comment:
these functions were not implemented at all. Maybe we should not somewhere that this still needs to be done? Or we throw an exception at this moment so that the use of this unimplement function is noticed.

Reply:
It could be that this will be changed completely. So I have no strong opinion for the moment. In future we will use realistic values stored in the CCDB, so this class might vanish completely...